modules/registrars/openprovider/openprovider.php

```diff
@@ -329,6 +330,50 @@ function openprovider_ResendIRTPVerificationEmail(array $params)
     return openprovider_registrar_launch_decorator('resendIRTPVerificationEmail', $params);
 }
 
+
+function openprovider_config_validate($params)
+{
+    $username = $params['Username'];
+    $password = $params['Password'];
+    $test = $params['test_mode'];
```

Comment:
better to use the variable name as it's (test_mode)

Reply:
This change has been added